CHANGELOG.md

```diff
@@ -15,6 +35,7 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- release 4.0.0 [\#49](https://github.com/voxpupuli/puppet-minecraft/pull/49) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I see you added the label to skip. Can you regenerate?

Reply:
oh ups, will do.